lib/colors.js

```diff
@@ -115,6 +115,13 @@ function applyTheme (theme) {
   for (var style in theme) {
     (function(style){
       colors[style] = function(str){
+        if (typeof theme[style] === 'object'){
```

Comment:
Shouldn't you be using `Array.isArray`? The value `{random: 12}` will still be passing this condition.